python/pyspark/ml/feature.py

```diff
@@ -512,6 +512,16 @@ class HashingTF(JavaTransformer, HasInputCol, HasOutputCol, HasNumFeatures, Java
     .. versionadded:: 1.3.0
     """
 
+    """
+    Binary toggle to control term frequency counts.
+    If true, all non-zero counts are set to 1.  This is useful for discrete probabilistic
+    models that model binary events rather than integer counts.
+    (default = False)
+    """
+    binary = Param(Params._dummy(), "binary",
+                   "Binary toggle to control term frequency counts",
+                   typeConverter=TypeConverters.toBoolean)
+
     @keyword_only
     def __init__(self, numFeatures=1 << 18, inputCol=None, outputCol=None):
```

Comment:
Thanks @yanboliang just updated the pull request with issues addressed. Let me know if there are  any other issues.